spec/cordova/platforms/platforms.spec.js

```diff
@@ -32,6 +32,29 @@ var PLATFORM_SYMLINK = path.join(os.tmpdir(), 'cordova_windows_symlink');
 
 shell.ln('-sf', PLATFORM_WITH_API, PLATFORM_SYMLINK);
 
+describe('platforms object', function () {
+    it('should have getPlatformApi function as a property', function () {
+        expect(platforms.getPlatformApi).toBeDefined();
+        expect(typeof platforms.getPlatformApi).toBe('function');
+    });
+
+    it('should have correct number of platform properties', function () {
+        expect(Object.keys(platforms).length).toBe(5);
+    });
+
+    it('should include the supported platforms', function () {
+        expect(platforms.android).toBeDefined();
+        expect(platforms.browser).toBeDefined();
+        expect(platforms.ios).toBeDefined();
+        expect(platforms.osx).toBeDefined();
+        expect(platforms.windows).toBeDefined();
+    });
+
+    it('should *not* include nonsense "www" platform (CB-14148)', function () {
+        expect(platforms.www).not.toBeDefined();
+    });
```

Comment:
I've been battling needlessly verbose tests. Would you be OK if we merge the last three tests to the following?
```js
it('should have all and only the supported platforms', function () {
    expect(Object.keys(platforms)).toEqual(jasmine.arrayWithExactContents([
        'android', 'browser', 'ios', 'osx', 'windows',
    ]));
});
```

Reply:
👍

Reply:
Do you want to incorporate that change?

Reply:
Yes I can incorporate the change later today. I would also like this PR to land with a commit message that will indicate what we did to the user in the RELEASENOTES, kinda like the way I updated the title.